### ISSUE TITLE
Bludit 3.7.1 / Docs X Theme

### ISSUE THREAD

**Issue by Markus-GS** [pytesNET/more-levels#2]
Hi,
with which templates is the plugin supposed to work?
I tried it on Bludit 3.7.1 with the Docs X template, but is is not working. When selecting a "level 2" Parent the new "level 3" entry afterwards is displayed at level 2 nevertheless.
So is this a problem with the plugin and 3.7.1?
Or does Docs X not support this functionatlity?
Thanks!

**Comment by SamBrishes** [comment]
Hellow,

this plugin currently just adds the functionallity to select each available Page as parent using the default Bludit administration theme "Booty". So, the current version doesn't support any (frontend) theme, but I'm currently working on a small update (which also adds some functions), and I will try to manipulate the respective frontend as far as possible. However, at the moment I'm not sure if it is possible to manipulate the output without to change / edit the theme files itself.

TLDR: Unfortunately, the current version requires you to adapt the theme on your own.

Sincerely,
Sam.
